frontend/src/widgets/HorizontalTableWidget.scss

```diff
@@ -27,15 +27,7 @@
     min-width: 90px;
   }
 
-  // Because putting the border on the sticky element doesn't work, and box-shadow
-  // isn't ideal because of z-indexing weirdness that happens when a ContextMenu
-  // is above the border.
-  .smarthub-horizontal-table-widget .smarthub-horizontal-table-last-column::before {
-    content: '';
-    position: absolute;
-    top: 0;
-    left: 0;
-    height: 100%;
+  .smarthub-horizontal-table-widget .usa-table .smarthub-horizontal-table-last-column {
     border-left: 1px solid black;
```

Comment:
I think you need to revert my bright idea here

Reply:
reverted here: [435aea4](https://github.com/HHS/Head-Start-TTADP/pull/2628/commits/435aea409e36c62876729f99d04b5b5fb4699e1b)